Gemfile.lock

```diff
@@ -58,9 +58,8 @@ GEM
       rdoc (>= 4.0.0)
       reline (>= 0.4.2)
     json (2.18.0)
-    json-schema (6.0.0)
+    json-schema (5.1.0)
       addressable (~> 2.8)
-      bigdecimal (~> 3.1)
```

Comment:
json-schema v6 only supports up to bigdecimal v3. In this change, we will prioritize to support bigdecimal v4 and proceed with bigdecimal v4 support for json-schema separately.